src/keylessBackup/KeylessBackupPhoneInput.tsx

```diff
@@ -197,7 +197,8 @@ const styles = StyleSheet.create({
     padding: Spacing.Thick24,
   },
   countryFlagStyle: {
-    backgroundColor: Colors.gray2,
-    marginRight: Spacing.Smallest8,
```

Comment:
I saw a change where a margin on phone number input changed from 16->24, is that just being made up here? If so, can we just leave it as is (to keep diffs from fork minimal)

Reply:
Without this rejiggering, the on-press shadow on the country selector doesn't respect the border radii on the right side and renders with right angles, strangely enough.